index.html

```diff
@@ -3254,38 +3316,46 @@ <h2>Processing a Manifest</h2>
 								result of running <a href="#processing-normalize">this step</a> on
 								<var>keyValue</var>.</p>
 						</li>
```

Comment:
Just to be precise: running this step on keyValue, lang, and dir. (ie, the input lang and dir as forwarded to the recursive call)

Reply:
Yes, and base. This is corrected in the latest commit.